libc/test/src/stdlib/SortingTest.h

```diff
@@ -66,45 +68,36 @@ class SortingTest : public LIBC_NAMESPACE::testing::Test {
     ASSERT_LE(array[24], 12310);
   }
 
-  void test_reversed_sorted_array(SortingRoutine sort_func) {
+  void test_reversed_sorted_array(SortFn sort_fn) {
```

Comment:
if you'd like to rename these variables that might fit better as a followup patch since it's not really related to this patch.

Reply:
Fair point, I had originally removed that parameter and then later added it again, should have gone with the same name. Will change.